src/sync/polling/updaters/splitChangesUpdater.ts

```diff
@@ -170,6 +206,8 @@ export function splitChangesUpdaterFactory(
               return Promise.resolve(!splitsEventEmitter.splitsArrived || (since !== splitChanges.till && (isClientSide || checkAllSegmentsExist(segments))))
```

Comment:
We can follow the same approach than Android SDK https://github.com/splitio/android-client/pull/550/files
 Remove `computeFromSets` function, and instead use the return value of `splits.removeSplits(mutation.removed)`:

```
          return Promise.all([
            splits.setChangeNumber(splitChanges.till),
            splits.addSplits(mutation.added),
            splits.removeSplits(mutation.removed),
            segments.registerSegments(mutation.segments)
          ]).then(([,addedList,removedList,]) => {
```